modules/data.land/NAMESPACE

```diff
@@ -1,10 +1,12 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(format,identifier)
```

Comment:
We've switched to naming functions with underscores rather than periods to avoid S3method misinterpretation. I'd recommend updating all function names and rerunning make document